components/brave_ads/common/BUILD.gn

```diff
@@ -5,26 +5,14 @@
 
 import("//mojo/public/tools/bindings/mojom.gni")
 
-mojom("mojom") {
-  sources = [ "brave_ads_host.mojom" ]
```

Comment:
do you need to delete `brave_ads_host.mojom`?

Reply:
done, deleted `brave_ads_host.mojom`